cmd/reconciler/main.go

```diff
@@ -100,6 +100,8 @@ var (
 	namespaceStrategy = flag.String(flags.namespaceStrategy, util.EnvString(reconcilermanager.NamespaceStrategy, ""),
 		fmt.Sprintf("Set the namespace strategy for the reconciler. Must be %s or %s. Default: %s.",
 			configsync.NamespaceStrategyImplicit, configsync.NamespaceStrategyExplicit, configsync.NamespaceStrategyImplicit))
+
+	nsControllerEnabled = flag.Bool("ns-controller-enabled", util.EnvBool(reconcilermanager.NSControllerEnabled, false), "")
```

Comment:
did you implement how this envvar/flag gets set? I can't find that code.

Reply:
It will be in a separate PR.
The envvar flag will be read from the NamespaceSelector mode, and set as RSync annotation. Then reconciler-manager sets the envvar by reading the annotation.